tests/unit_tests/test_plans.py

```diff
@@ -91,7 +92,7 @@ def test_spectroscopy(RE: RunEngine):
     }
```

Comment:
Do we want to assert that prepare was invoked on the detector with such and such argument?

Reply:
Yes, or assert that `detector.drv.acquire_time` is a particular value